app/js/app/directives/equation_editor/EquationEditor.js

```diff
@@ -3,47 +3,51 @@ define(function(require) {
 
     var MySketch = require("inequality").MySketch;
 
-	return ["$timeout", "$rootScope", "api", function($timeout, $rootScope, api) {
+    return ["$timeout", "$rootScope", "api", function($timeout, $rootScope, api) {
 
-		return {
-			scope: true,
-			restrict: "A",
-			templateUrl: "/partials/equation_editor/equation_editor.html",
-			link: function(scope, element, attrs) {
+        return {
+            scope: true,
+            restrict: "A",
+            templateUrl: "/partials/equation_editor/equation_editor.html",
+            link: function(scope, element, attrs) {
 
                 element.on("touchstart touchmove", "canvas", function(e) {
                     e.preventDefault();
                 });
 
                 var sketch = null;
 
-                scope.canvasOffset = { };
+                scope.canvasOffset = {};
                 scope.draggingNewSymbol = false;
-
                 scope.equationEditorElement = element;
 
                 scope.selectedSymbols = [];
                 scope.selectionHandleFlags = {
                     showCalc: false,
                     showResize: true,
-                    showMove: true
+                    showMove: false
                 };
 
                 scope.$on("triggerCloseMenus", function() {
-                	scope.$broadcast("closeMenus");
+                    scope.$broadcast("closeMenus");
                 });
 
                 scope.$on("triggerResizeMenu", function() {
-                	scope.$broadcast("resizeMenu");
+                    scope.$broadcast("resizeMenu");
                 });
 
                 $(window).on("resize", function() {
-                    element.find(".top-menu").css({"bottom": scope.equationEditorElement.height()}).removeClass("active-menu");
+                    element.find(".top-menu").css({
+                        "bottom": scope.equationEditorElement.height()
+                    }).removeClass("active-menu");
                 });
 
                 scope.$on("newSymbolDrag", function(_, symbol, pageX, pageY, mousePageX, mousePageY) {
-                    scope.draggingNewSymbol = true;
 
+                    console.debug(scope.dimmensions);
```

Comment:
This `console.debug(...)` needs to go. I hope dimensions was spelt correctly elsewhere? :P